### PR TITLE
Merge master HEAD into openj9-staging

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Resolved conflicts:
```
	src/java.base/share/classes/java/lang/foreign/VaList.java
	src/java.base/share/classes/jdk/internal/foreign/CABI.java
	src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java
	src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java
	src/java.base/share/classes/jdk/internal/foreign/abi/SharedUtils.java
	src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/CallArranger.java
```

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by keithc-ca** [comment]
In the files I've asked you to change, can you please arrange that changes for openj9 are only additions to imports, not reordering of any those that come from upstream (even though in some cases, those are not sorted).

File: src/java.base/share/classes/java/lang/foreign/VaList.java [hunk 1]
```diff
@@ -41,6 +41,7 @@
 import jdk.internal.foreign.abi.aarch64.macos.MacOsAArch64VaList;
 import jdk.internal.foreign.abi.ppc64.aix.AixPPC64VaList;
 import jdk.internal.foreign.abi.ppc64.sysv.SysVPPC64leVaList;
+import jdk.internal.foreign.abi.aarch64.windows.WindowsAArch64VaList;
```
[the remark this answers is not in the document]
Comment (JasonFengJ9) [reply src/java.base/share/classes/java/lang/foreign/VaList.java]: Moved

File: src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java [hunk 2]
```diff
@@ -36,6 +36,7 @@
 import jdk.internal.foreign.abi.aarch64.macos.MacOsAArch64Linker;
 import jdk.internal.foreign.abi.ppc64.aix.AixPPC64Linker;
 import jdk.internal.foreign.abi.ppc64.sysv.SysVPPC64leLinker;
+import jdk.internal.foreign.abi.aarch64.windows.WindowsAArch64Linker;
```
[the remark this answers is not in the document]
Comment (JasonFengJ9) [reply src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java]: Moved

File: src/java.base/share/classes/jdk/internal/foreign/abi/SharedUtils.java [hunk 3]
```diff
@@ -39,6 +39,7 @@
 import jdk.internal.foreign.abi.aarch64.macos.MacOsAArch64Linker;
 import jdk.internal.foreign.abi.ppc64.aix.AixPPC64Linker;
 import jdk.internal.foreign.abi.ppc64.sysv.SysVPPC64leLinker;
+import jdk.internal.foreign.abi.aarch64.windows.WindowsAArch64Linker;
```
[the remark this answers is not in the document]
Comment (JasonFengJ9) [reply src/java.base/share/classes/jdk/internal/foreign/abi/SharedUtils.java]: Moved

**Comment by keithc-ca** [comment]
Please fix the imports in
* `src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java`
* `src/java.base/share/classes/jdk/internal/foreign/abi/SharedUtils.java`

so they don't reorder anything from upstream.

**Comment by JasonFengJ9** [comment]
Restored upstream format, the change was made in https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/551

**Comment by keithc-ca** [comment]
> the change was made in #551

Yes, I probably should have asked for that change not be made there. Thanks for fixing it here.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**